addon/components/feature-controls.js

```diff
@@ -1,102 +1,123 @@
+/* eslint-disable ember/no-classic-components */
 import Component from "@ember/component"
 import layout from "../templates/components/feature-controls"
 import { inject as service } from "@ember/service"
 import { action, set } from "@ember/object"
 import { assign } from "@ember/polyfills"
 import config from "ember-get-config"
 import windowUtil from "ember-feature-controls/utils/window"
+import classic from "ember-classic-decorator"
 
-const { featureFlags, featureControls } = config
-
-export default Component.extend({
-  layout,
-  tagName: '',
-  features: service(),
-  featureControlStorage: service(),
-  showRefresh: true,
-  showReset: true,
+const {
   featureControls,
   featureFlags,
+} = config
+
+@classic
```

Comment:
Why do you need  `@classic` decorator here ?

Reply:
Oh I just saw the `init` ... Why not using `did-render` modifier to fire the `refresh` method ?

Reply:
Yes, because the component is still "shaped as a Classic component".

I could have gone further yes, modifier, Glimmer, but I was afraid it would be a change too big, involving potential breaking change in the applications using that add-on. I rather wanted a simple first step here.